src/aad/getMemberDetails.ts

```diff
@@ -27,7 +28,7 @@ export const getMemberDetails = async (): Promise<IMemberDetails[]> => {
 
   const promiseArray = groupIds.map(async (groupId) => {
     const requestUrl = new URL(
-      `/v1.0/groups/${groupId.trim()}/members?$count=true&$top=999&$filter=accountEnabled eq true`,
+      `/v1.0/groups/${groupId.trim()}/members?$count=true&$top=${config.aad.membersToRequest}&$filter=accountEnabled eq true`,
```

Comment:
where is the config updated?

Reply:
This value has weirdly _always_ been in our config object. It defaults to 999 anyway if not provided, which is what we'll use in practice, so shouldnt need any updates in terraform until we actually want to give it a different value.

It can be overridden directly for testing purposes.

Reply:
lmao